src/main/scala/io/github/nyango/formatconverter/FormatConverter.scala

```diff
@@ -8,8 +8,17 @@ object FormatConverter {
     yamlValueToJsValue(yamlStr.parseYaml)
   }
 
-  def jsValueToYamlStr(json: JsValue): String = {
-    jsValueToYamlValue(json).prettyPrint
+  def jsValueToYamlStr(
+      json: JsValue,
+      flowStyle: FlowStyle = FlowStyle.DEFAULT,
+      scalarStyle: ScalarStyle = ScalarStyle.DEFAULT,
+      lineBreak: LineBreak = LineBreak.DEFAULT
```

Comment:
This is due to my format setting of [IntelliJ](https://www.jetbrains.com/idea/) 2017.1 used in the team.
Sorry, I cannot explain why.